www/gatsby-config.js

```diff
@@ -214,6 +215,7 @@ module.exports = {
               {
                 allMdx(
                   sort: { order: DESC, fields: [frontmatter___date] }
+                  limit: 10,
```

Comment:
Is this intentional?

Reply:
Yeah, rss feeds typically only show the ten newest posts and rendering the MDX here takes a long time as it's rendering every blog post.